Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Tower/FanSpeed/ReturnWaterTemperature/Subsequences/Enable.mo

```diff
@@ -0,0 +1,390 @@
+﻿within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Tower.FanSpeed.ReturnWaterTemperature.Subsequences;
+block Enable "Sequence for enabling and disabling tower fan"
```

Comment:
This is currently implemented for CWRT only? Just noticing that at some point it should have a flag to distinguish between when tower speed is controlled based on CWRT or CWST and be pulled up to a higher level in the package structure.

Reply:
Yes, this is just for CWRT. In fact, this sequence can be used for the control based on CWST. So I would prefer to leave (`TTow` and `TTowSet`) it like that for now.

Reply:
Ok, it might be good to have that documented somewhere, either in the comment or in the info section just for future reference.